datashader/glyphs.py

```diff
@@ -66,3 +67,182 @@ def _compute_x_bounds(self, df):
 
     def _compute_y_bounds(self, df):
         return df[self.y].min(), df[self.y].max()
+
+
+class Line(Glyph):
+    """A point, with points defined by ``x`` and ``y``.
+
```

Comment:
A polyline, i.e., a list of connected line segments, with vertexes defined by ``x`` and ``y``.

(but if that's correct, seems like this should be called Polyline or PolyLine.  To me, Line implies Straight Line, and this is more like a piecewise-linear approximation to a curve.)